e2etest/newe2e_resource_manager_azstorage.go

```diff
@@ -30,47 +30,65 @@ func (acct *AzureAccountResourceManager) ApplySAS(URI string, loc common.Locatio
 		return URI
 	}
 
-	sasVals := opts.AzureOpts.SASValues
-	if sasVals == nil {
-
+	var sasVals GenericServiceSignatureValues
+	if opts.AzureOpts.SASValues == nil {
 		sasVals = GenericServiceSignatureValues{}
+	} else {
+		sasVals = opts.AzureOpts.SASValues.(GenericServiceSignatureValues)
```

Comment:
What if we get Account SAS values?

Reply:
Good point, we might need to add a GenericAccountSignatureValues type that keeps track of that